### PR TITLE
Fix native typography types

### PR DESCRIPTION
**Pull request by iryanclarke**
Since moving the Typography tokens under the main theme file, there was some improper formatting generated for the Native build!

We use the `font-weight-to-string` and  `rm-px` to help properly format the old typography tokens, but because these new `aviaryTypography` was part of a different type of object, these formatters were not properly being applied!

So, this MR adds in the proper parsing/formatting to ensure that these values are properly generated for use in the `fs-native` repository!

### CODE REVIEW

File: src/helpers/style-dictionary/transforms/transforms.js [hunk 1]
```diff
@@ -58,3 +58,24 @@ StyleDictionary.registerTransform({
     return token.value.toString();
   },
 });
+
+/* Converts typography attributes to
+ * proper types to be used in react native
+ *
+ * fontWeight: number => string
+ * fontSize: string => number
+ */
```
Comment (CCAyl) [inline src/helpers/style-dictionary/transforms/transforms.js:67]: yaas +1 to helpful comments in these crazy files

**Review by CCAyl — APPROVED** [review]
nice, lgtm 🚀

**Comment by alexandra-lim** [comment]
Looking good! Only thing -- can we update the version number in package.json?

**Review by alexandra-lim — APPROVED** [review]

**Comment by iryanclarke** [comment]
> Looking good! Only thing -- can we update the version number in package.json?

Definitely! We should really tweak the workflow here, and do version bumps automatically on release

**Review by bambie1 — APPROVED** [review]

**Pull request merged**